### DISCUSSION TITLE
Number Of Infections In Syria Not Added

### DISCUSSION THREAD

**Discussion by ghost** [neherlab/covid19_scenarios#429] [General]
This is the first issue i open in GitHub,So
First,There is over 15 infected,While 2 are saved and 2 deaths,With total of 19
I don't know if this is useful,And if so,where can i contribute to add them?

**Comment by noleti** [comment]
Hi @Rabios, we are already providing [case data](https://github.com/neherlab/covid19_scenarios/blob/master/data/case-counts/ecdc/Syrian%20Arab%20Republic.tsv) for Syria via the  https://www.ecdc.europa.eu/en API. If you have better data sources, you would ideally write a parser for that sources (instructions are in `data/README.md`). You can select Syria as source in the 'confirmed cases' field in the app. But we don't have a scenario for Syria yet, because we don't have information on the Syrian population size, number of hospital beds, etc. If you have such data, it would be great if you could create a Pull Request with additions made to https://github.com/neherlab/covid19_scenarios/blob/master/data/populationData.tsv

**Reply by ghost** [reply]
In fact I'm syrian,So the info in the issue are the better which provided by the syrian government

But yes,I would like to help

Best wishes,rabia
